### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by ihostage**

### CODE REVIEW

**Comment by ihostage** [comment]
Hi, @leleuj! 👋 
Can you enable Travis for this repository? 😄 
And also approve access for Renovate 😄

**Comment by leleuj** [comment]
@ihostage I will enable Travis next week.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: java
+
+jdk:
+  - openjdk8
```
Comment (leleuj) [inline .travis.yml]: For pac4j, I use: `oraclejdk8`but I don't think it's an issue...

Comment (ihostage) [reply .travis.yml]: On all our repository Travis fail after Oracle License is changed 😞And now I automatically change `oraclejdk8` to `openjdk8` in all `.travis.yml` files that I touch 😂

**Comment by ihostage** [comment]
> @ihostage I will enable Travis next week.

👍

**Pull request merged**